src/plugins/chart_expressions/expression_metric/public/components/metric.scss

```diff
@@ -6,13 +6,15 @@
 //   mtrChart__legend-isLoading
 
 .mtrVis {
+  @include euiScrollBar;
```

Comment:
Why do we need these changes? I don't notice any difference

Reply:
Because we use `VisualizationContainer` here with styles which includes styles for scrollbar, but styles of `VisualizationContainer` loaded only when we open visualization first (because we load styles here src\plugins\visualizations\public\vis_async.ts). If we open metric directly in canvas you will see so that we don't have scroll at all because we don't load styles for `VisualizationContainer`. So to be sure that we will have correct scroll I added these styles here.